hw/ip/i2c/README.md

```diff
@@ -14,10 +14,10 @@ See that document for integration overview within the broader top level system.
 - Support for I2C Host ("I2C Master"<sup>1</sup>) and I2C Target ("I2C Slave"<sup>1</sup>) device modes
 - Support for standard (100 kbaud), fast (400 kbaud) and fast-plus (1 Mbaud) modes
```

Comment:
The spec uses capitalized words here (e.g. `Standard-mode, Fast-mode, Fast-mode Plus, Fm+, `) so I think we should mirror that.

Reply:
Thanks, should be resolved now.